conf/config.yaml.sample

```diff
@@ -58,17 +58,46 @@ cortx:
     auth_admin: sgiamadmin
     auth_secret: s3_auth_admin_secret
   motr:
-    client_instances: 0
     interface_type: tcp
     interface_family: inet
     transport_type: libfab
     ios:
       endpoints:
-      - motr://data1-node1.cortx-cluster.com:3232
+      - inet:tcp:data-node1@28002
+      - inet:tcp:data-node1@28003
+      - inet:tcp:data-node2@28002
+      - inet:tcp:data-node2@28003
+      - inet:tcp:data-node3@28002
+      - inet:tcp:data-node3@28003
+    confd:
+      endpoints:
+      - inet:tcp:data-node1@28004
+      - inet:tcp:data-node2@28004
+      - inet:tcp:data-node2@28004
+    s3_instances: 1 # cortx s3 servers
+    s3:
+      endpoints:
+      - inet:tcp:server-node1@28002
+      - inet:tcp:server-node1@28003
+      - inet:tcp:server-node2@28002
+      - inet:tcp:server-node2@28003
+      - inet:tcp:server-node3@28002
+      - inet:tcp:server-node3@28003
+    client_instances: 1 # cortx clients, Optional
+    client:
+      endpoints:
+      - inet:tcp:data-node1@28005
+      - inet:tcp:data-node2@28005
+      - inet:tcp:data-node3@28005
   hare:
     hax:
       endpoints:
-      - tcp://hax.seagate.com:28000
+      - inet:tcp:data-node1@28001
+      - inet:tcp:data-node2@28001
+      - inet:tcp:data-node3@28001
+      - inet:tcp:server-node1@28001
+      - inet:tcp:server-node2@28001
+      - inet:tcp:server-node3@28001
```

Comment:
@mssawant : Can you add the reference for endpoints for using hctl status using curl with http and https and corresponding ports ?

Reply:
@shriya-deshmukh : Can you add following endpoint under Hax, they are need to communicate with Hax http server by CSM.

http://data-node1@28006
https://data-node1@28007
http://data-node2@28006
https://data-node2@28007
http://data-node3@28006
https://data-node3@28007
http://server-node1@28006
https://server-node1@28007
http://server-node2@28006
https://server-node2@28007
http://server-node3@28006
https://server-node3@28007

Reply:
During the discussion with @kupranay , he suggested that better option i.e. a cluster-ip service for Hax https service can be added, and then all hax processes can start the internal http server on the same port. Also we will use only https protocol for it.
@mssawant  @ujjwalpl : What is you opinion about it?

Reply:
Endpoints in the solution need to be in the URL format for consistency across all kinds of endpoints.
Example: tcp://server1-node1:32320/<if anything else>

More examples at
https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/611615649/CORTX+Configuration+Parameters

Reply:
Done